web_external/js/collections/SourceCollection.js

```diff
@@ -0,0 +1,6 @@
+minerva.collections.SourceCollection = minerva.collections.MinervaCollection.extend({
+
+    model: minerva.models.SourceModel,
+    path: 'minerva_source'
+
+});
```

Comment:
Should this conditionally set model to be minerva.models.WmsSourceModel (similar to the way DataSetCollection might return a WmsDatasetModel or S3DatasetModel depending on attrs.meta.minerva.dataset_type)?